src/pages/Improbability/Improbability.component.tsx

```diff
@@ -0,0 +1,69 @@
+import Typography from '@mui/material/Typography';
+import { useForm } from 'react-hook-form';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
```

Comment:
Please use this button - `src/components/Button/Button.component.tsx`. It's just a mui button with a loading state that might be helpful in this page in the future